components/Advocates/MeetTheAdvocates/AdvocatesMeetTheAdvocatesCard.vue

```diff
@@ -3,7 +3,7 @@
     class="advocates-meet-the-advocates-card"
     :image="image"
     :title="name"
-    :tags="formattedRegion"
+    :primary-tag="formattedRegion"
```

Comment:
I think we can continue having this as an array, we are not sure if we can have a list of `primary-tags` in the future

Reply:
updated here https://github.com/Qiskit/qiskit.org/pull/3442/commits/80a62b8e218f9e1fc38748f1d4625a9cf8fe141d